cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -962,16 +982,8 @@ void initGeoFences() {
                             for (int i = 0; i < geoFenceArray.length(); i++) {
                                 JSONObject geoFence = geoFenceArray.getJSONObject(i);
                                 if (geoFence != null) {
-                                    geofenceList.add(new Geofence.Builder()
-                                            .setRequestId(geoFence.getString("_id"))
-                                            .setCircularRegion(
-                                                    geoFence.getDouble("latitude"),
-                                                    geoFence.getDouble("longitude"),
-                                                    geoFence.getLong("radius")
-                                            )
-                                            .setExpirationDuration(Geofence.NEVER_EXPIRE) // Future: use "endsAt" instead
-                                            .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_ENTER | Geofence.GEOFENCE_TRANSITION_EXIT)
-                                            .build());
+                                    geofenceList.add(new Geofence.Builder().setRequestId(geoFence.getString("_id")).setCircularRegion(geoFence.getDouble("latitude"), geoFence.getDouble("longitude"), geoFence.getLong("radius")).setExpirationDuration(Geofence.NEVER_EXPIRE) // Future: use "endsAt" instead
```

Comment:
revert formatting
here and all spots below where it got messed up

Reply:
okay reformatted